bourbon.gemspec

```diff
@@ -21,12 +21,12 @@ meaning they should be as close to the original CSS syntax as possible.
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency("sass", "~> 3.4")
-  s.add_runtime_dependency("thor")
+  s.add_runtime_dependency("sass", "~> 3.4.15")
+  s.add_runtime_dependency("thor", "~> 0.19.1")
 
   s.add_development_dependency("aruba", "~> 0.6.2")
-  s.add_development_dependency("css_parser")
-  s.add_development_dependency("rake")
-  s.add_development_dependency("rspec")
+  s.add_development_dependency("css_parser", "~> 1.3.6")
+  s.add_development_dependency("rake", "~> 10.4.2")
+  s.add_development_dependency("rspec", "~> 3.3.0")
```

Comment:
Do we need to lock all this down to patch versions? I know for `aruba` specifically we did, but we had specific issues with higher versions. If we can, I'd lock them on the minor version instead.